src/pow.cpp

```diff
@@ -428,6 +432,8 @@ RCP<const Basic> pow_expand(const RCP<const Pow> &self)
         }
     }
     RCP<const Basic> result = Add::from_dict(add_overall_coeff, std::move(rd));
+    RCP<const Number> i = integer(-1);
+    if(negative_pow) result = rcp(new Pow(result, i));
```

Comment:
This case won't throw an error as the base isn't an instance of Add. expand() will immediately return self in this case. However, I understand what you mean. I'll make the change.